common/src/main/java/dev/latvian/mods/itemfilters/item/TagFilterItem.java

```diff
@@ -81,16 +76,6 @@ public Collection<StringValueFilterVariant> getValueVariants(ItemStack filter) {
 		}).toList();
 	}
 
-	@Override
-	public void getDisplayItemStacks(ItemStack filter, List<ItemStack> list) {
-		NonNullList<ItemStack> list1 = NonNullList.create();
-
-		StreamSupport.stream(Registry.ITEM.getTagOrEmpty(TagKey.create(Registry.ITEM_REGISTRY, new ResourceLocation(getValue(filter)))).spliterator(), false)
-				.forEach(e -> e.value().fillItemCategory(CreativeModeTab.TAB_SEARCH, list1));
-
-		list.addAll(list1);
-	}
-
```

Comment:
Since tags are reset and rebound on server reload, wouldn't it make sense to keep a custom cache for them (or to just forfeit caching altogether in this place, since the prefiltered list exists in vanilla already)?

As it stands right now, a tag filter cached by Item Filters itself would keep its out of date values across reloads until it is booted from the cache

Reply:
Yeah, good point, I'd forgotten about tags changing.  I'll have a think about the best way of handling this (might be necessary to clear the whole cache, since tag filters can be embedded in other filters...)

Reply:
OK, added cache clearing (server and non-local clients) when server-side resources are reloaded